xclim/sdba/_adjustment.py

```diff
@@ -0,0 +1,177 @@
+"""
+Adjustment algorithms.
+
+This file defines the different steps, to be wrapped into the Adjustment objects.
+"""
+import numpy as np
+import xarray as xr
+
+from . import nbutils as nbu
+from . import utils as u
+from .base import map_blocks, map_groups
+
+
+@map_groups(
+    af=["<PROP>", "quantiles"], hist_q=["<PROP>", "quantiles"], scaling=["<PROP>"]
+)
+def dqm_train(ds, *, dim, kind, quantiles):
```

Comment:
Are these public or private methods ? If public, I suggest writing complete docstrings, otherwise prepend name with _.

Reply:
Hum, the entire module is private, right ? So no need for full docstrings.